File: problemSet1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int n, int h);

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    for (int i = 1; i <= h; i++)
    {
        print_row(i, h);
    }
}


void print_row(int n, int h)
{
    int space = h - n;
    for (int i = 0; i < space; i++)
    {
        printf(" ");
    }
    for (int i = 0; i < n; i++)
    {
        printf("#");
    }

    printf("  ");

    for (int i = 0; i < n; i++)
    {
        printf("#");
    }

    printf("\n");
}

File: problemSet6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if (h > 0 and h <= 8):
        break

brick = "#"

for i in range(1, h+1):
    print(f"{' '*(h-i)}{brick*i}  {brick*i}")
